src/applications/disability-benefits/686c-674/config/helpers.js

```diff
@@ -71,3 +71,91 @@ export const cityTitle = (
     City <strong>or</strong> county
   </>
 );
+
+export const isInsideListLoopReturn = (
+  chapter,
+  outerField,
+  isInsideListLoop,
```

Comment:
I think `isInsideListLoop` return can be removed from both this function and the one below. It appears this parameter isn't used in either of them.

Reply:
yep, just removed this